samples/apiProjects/SampleSOAPApi/README.md

```diff
@@ -0,0 +1,32 @@
+# WSO2 API Manager Project
+
```

Comment:
add a description saying what this is

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/2972/commits/ca3cab03cae43525f286b71c064fac30bdc43bcf